src/filelock/_api.py

```diff
@@ -169,6 +172,11 @@ def timeout(self, value: float | str) -> None:
         """
         self._context.timeout = float(value)
 
+    @property
+    def mode(self) -> int:
```

Comment:
note: we add this as a property to allow subclasses to implement there own context management.